doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -46,9 +47,9 @@ A job failure might be caused by one or more of the following reasons:
 
 * permutation of config values: for instance, `qa/suites/rados/thrash
```

Comment:
Neha, I think that these two bullet points need to be rewritten. "environment setup" and "permutation of config values" don't sound to my ear like causes of failure. Maybe the document would be better if it said "improper environment setup" and "incorrect config values". Let me know what you think.

Also, if you could explain a little about what each of these bullet points means to refer to, I'd be grateful and I'd probably be able to put together a nice two-item list.

Reply:
we discussed this offline